client/tests/integration/roles.rs

```diff
@@ -137,3 +137,33 @@ fn unregistered_role_removed_from_account() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn role_with_invalid_permissions_is_not_accepted() -> Result<()> {
+    let (_rt, _peer, test_client) = <PeerBuilder>::new().with_port(10_705).start_with_runtime();
```

Comment:
It's been used in `unregistered_role_removed_from_account` test it seems. I know it's frustrating, but I usually just grep for the port through the repo to see if it's mentioned anywhere.